src/utils/cplay.c

```diff
@@ -492,12 +495,97 @@ void get_codec_aac(FILE *file, struct compr_config *config,
 	codec->format = format;
 }
 
+
+static int skip_id3v2_header(FILE *file)
```

Comment:
Remove this new line. As we already have one above.

Reply:
Should be ok now, thanks